module/gnb/api/procedure.go

```diff
@@ -335,6 +335,172 @@ func DeRegistration(ue *RanUeContext) error {
 
 }
 
+// TestGUTIRegistration implements a function registartion using the GUTI after using SUPI reg
```

Comment:
To add to function description to dismiss misusage: "Registration GUTI is to use after a successfully registered procedure using SUPI and a successful de-registration procedure"